doc: add exporting backing image details (#1071)
longhorn/website: content/docs/1.7.0/advanced-resources/backing-image/backing-image.md

4. Restore a backing image from the backupstore, For more information, see [Backing Image Backup](../backing-image-backup).
5. Clone a backing image.

> #### How Exporting Volume Works
> A backing image serves as the initial snapshot in the snapshot chain of a Longhorn volume. When export a backing image from an existing Longhorn volume, the volume may already have an associated backing image. If a volume with a backing image is exported, the process commits the original backing image and merges it with the delta changes, resulting in a new consolidated backing image.

Reviewer: ```suggestion
#### Volume exporting

A backing image serves as the initial snapshot in the snapshot chain of a Longhorn volume. When you export a volume with an associated backing image, Longhorn merges that image with the delta changes, resulting in a new consolidated backing image.
```

Author: fixed.
